alvr/session/src/settings.rs

```diff
@@ -585,8 +585,9 @@ pub struct ConnectionDesc {
     #[schema(advanced)]
     pub on_disconnect_script: String,
 
-    #[schema(advanced)]
-    pub enable_fec: bool,
+    // Max packet size is 64KB for TCP and 65507 bytes for UDP
```

Comment:
Where did you get the upper limit for TCP and UDP? Isn't it a bit too large for UDP?

Reply:
I found UDP packet size here, and during my experimentations, I found it's actually the max size that can be sent by socket:
https://erg.abdn.ac.uk/users/gorry/course/inet-pages/udp.html#:~:text=A%20UDP%20datagram%20is%20carried,and%2065%2C527%20bytes%20for%20IPv6.

For TCP, it's a limitation because TCP header size is 16-bit, but technically you can send packet of any size and it will be fragmented automatically.

Reply:
I think some routers need a much lower value. u16::MAX is the UDP jumbo packet side but some network tend to drop them. instead a good value is 1200 bytes.

Reply:
Jumbo packets are not even reaching u16::MAX, but fragmentation does its thing. But to be honest, I dunno if it's worth sending packets larger than MTU.

Just in case, the option right now is unused, and packet size is hardcoded to the value previously used:
https://github.com/alvr-org/ALVR/pull/1344/files#diff-17f1a190fe3294b8c510338673a54b75c5f26f7059771714924cdcaeb2a2b9ffR112